src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1693,6 +1693,13 @@ public override BoundNode VisitFieldAccess(BoundFieldAccess node)
 
         private void VisitFieldAccessInternal(BoundExpression receiverOpt, FieldSymbol fieldSymbol)
         {
+            // VisitLvalue and VisitRValue don't accept null arguments, and if receiverOpt is null
+            // then there's nothing left to visit anyway.
+            if (receiverOpt == null)
```

Comment:
Also we need to think if we should visit ```nameof``` argument at all, it is not being evaluated. CC @gafter.

---
In reply to: [80124110](https://github.com/dotnet/roslyn/pull/13941#discussion_r80124110) [](ancestors = 80124110)